templates/includes/sidebar.html

```diff
@@ -0,0 +1,17 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="#">Home page</a>
+  </li>
+
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+
+  <li>
+    <a href="#">Cars</a>
+  </li>
+
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The href attributes in the anchor tags are currently set to `#`, which is a placeholder. Consider replacing these with actual URLs or Django URL template tags to enable proper navigation.